class/defaults.yml

```diff
@@ -13,12 +13,18 @@ parameters:
 
     ignore_alerts: []
 
+    # secrets exporter configs
     include_namespaces: []
-    exclude_namespaces:
-      - openshift-operator-lifecycle-manager
```

Comment:
What is the reason for this?

Reply:
I figured we should manage these params in the commodore defaults.
Mainly bc it should be the same over all distributions and we don't wan't to manage this per cluster.
We should however have a discussion on the commodore defaults an what should go in there.

Reply:
Sounds reasonable.